src/graphic/minimap_renderer.h

```diff
@@ -39,6 +39,8 @@ enum class MiniMapLayer {
 	ViewWindow = 64,
 	StartingPositions = 128,
 	Ship = 256,
+	Traffic = 512,
+	Artifacts = 1024,
```

Comment:
Those are becoming unreadable ;)
When you have so many constants please use bitshift notation:
```c++
 1 = 1 << 0
 2 = 1 << 1
 4 = 1 << 2
 8 = 1 << 3
16 = 1 << 4
...
```

Reply:
ok good idea